saleor/graphql/translations/mutations/attribute_value_bulk_translate.py

```diff
@@ -0,0 +1,133 @@
+        description="Attribute value ID.",
+    )
+    external_reference = graphene.String(
+        required=False,
+        description="External reference of an attribute value.",
+    )
+    language_code = LanguageCodeEnum(
+        required=True, description="Translation language code."
+    )
+    translation_fields = AttributeValueTranslationInput(
+        required=True,
+        description="Translation fields.",
+    )
+
+    class Meta:
+        doc_category = DOC_CATEGORY_ATTRIBUTES
+
+
+class AttributeValueBulkTranslate(BaseBulkTranslateMutation):
+    count = graphene.Int(
+        required=True,
+        description="Returns how many translations were created/updated.",
+    )
+    results = NonNullList(
+        AttributeValueBulkTranslateResult,
+        required=True,
+        default_value=[],
+        description="List of the translations.",
+    )
+
+    class Arguments(BaseBulkTranslateMutation.Arguments):
+        translations = NonNullList(
+            AttributeValueBulkTranslateInput,
+            required=True,
+            description="List of attribute values translations.",
+        )
+
+    class Meta:
+        description = (
+            "Creates/updates translations for attributes values."
+            + ADDED_IN_314
+            + PREVIEW_FEATURE
+        )
+        base_model = models.AttributeValue
+        translation_model = models.AttributeValueTranslation
+        translation_fields = ["name", "rich_text", "plain_text"]
+        base_model_relation_field = "attribute_value"
+        result_type = AttributeValueBulkTranslateResult
+        error_type_class = TranslationBulkError
+        permissions = (SitePermissions.MANAGE_TRANSLATIONS,)
+        doc_category = DOC_CATEGORY_ATTRIBUTES
+
+    @classmethod
+    def clean_translations(cls, data_inputs, index_error_map):
+        cleaned_inputs_map: dict = {}
+
+        for index, data in enumerate(data_inputs):
+            id = data.get("id")
+            external_ref = data.get("external_reference")
+
+            try:
+                validate_one_of_args_is_in_mutation(
+                    "id",
+                    id,
+                    "external_reference",
+                    external_ref,
+                    use_camel_case=True,
+                )
+            except ValidationError as exc:
+                index_error_map[index].append(
+                    TranslationBulkError(
+                        message=exc.message,
+                        code=TranslationErrorCode.INVALID.value,
+                    )
+                )
+                cleaned_inputs_map[index] = None
+                continue
+
+            if id:
+                data["id"] = graphene.Node.from_global_id(id)[1]
+            cleaned_inputs_map[index] = data
+
+        return cleaned_inputs_map
+
+    @classmethod
+    def pre_update_or_create(cls, instance, input_data):
+        if "name" not in input_data.keys() or input_data["name"] is None:
```

Comment:
Why not `if not input_data.get("name"):` or `if input_data.get("name") is None:` (if you accept empty strings)?